Makefile

```diff
@@ -1,10 +1,11 @@
-SHELL=/bin/bash -o pipefail
-GIT_COMMIT:=$(shell git rev-parse --short HEAD)
-LDFLAGS:=-X main.GitCommit=${GIT_COMMIT}
-GIT_UNTRACKED=$(shell git diff-index --quiet HEAD -- || echo "(updated)")
-LDFLAGS+=-X main.GitUntracked=${GIT_UNTRACKED}
-GIT_TAG=$(shell git describe)
-LDFLAGS+=-X main.GitLatestTag=${GIT_TAG}
+SHELL           = /bin/bash -o pipefail
+GIT_COMMIT     := $(shell git rev-parse --short HEAD)
+LDFLAGS        := -X main.GitCommit=${GIT_COMMIT}
+GIT_UNTRACKED   = $(shell git diff-index --quiet HEAD -- || echo "(updated)")
+LDFLAGS        += -X main.GitUntracked=${GIT_UNTRACKED}
+GIT_TAG         = $(shell git describe)
+LDFLAGS        += -X main.GitLatestTag=${GIT_TAG}
+REBUILD_DOCS    ?= true
```

Comment:
@lyuboxa why was this necessary?

Reply:
AFACIT, this should be `"true"` to make sure we still can automatically generate documentation.

Reply:
Opened PR to fix and add documentation https://github.com/meroxa/cli/pull/415

Reply:
`?= true` should default to true unless it's set, so I don't think there should be a net change in behavior unless someone explicitly sets it to false.

Reply:
@ahmeroxa AFAICT, it's not the same as `"true"` https://github.com/meroxa/cli/pull/413/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R28. So default behavior is "don't" build docs.